javascript/static/main.js

```diff
@@ -1,4 +1,4 @@
-import objects from './objects/*.yml'  // loads all the yaml objects and parses them.
+import objects from '../../objects/*.yml'  // loads all the yaml objects and parses them.
```

Comment:
Cool, I was afraid to do this because it was venturing outside the `static`  directory and other static file servers I've used have refused to do this.